icubam/map_builder.py

```diff
@@ -1,8 +1,7 @@
-from absl import logging
```

Comment:
This has potential side effect, maybe add `# noqa` for now?

Reply:
I think @dulacarnold asserted that it did not have side effects?

Reply:
> I think @dulacarnold asserted that it did not have side effects?

It does change the logging handler, see https://github.com/abseil/abseil-py/blob/06edd9c20592cec39178b94240b5e86f32e19768/absl/logging/__init__.py#L1167

but confirmation needed indeed.

Reply:
Maybe best to leave it in then, otherwise we might get inconsistent logging.